inbm/dispatcher-agent/fpm-template/usr/share/dispatcher-agent/manifest_schema.xsd

```diff
@@ -487,7 +492,7 @@
                 <xs:element name="fetch" type="URLMax700Chars" minOccurs="0" maxOccurs="1"/>
                 <xs:element name="signature" type="SigMax1000Chars" minOccurs="0" maxOccurs="1"/>
                 <xs:element name="username" type="Max50Chars" minOccurs="0" maxOccurs="1"/>
-                <xs:element name="password" type="Max50Chars" minOccurs="0" maxOccurs="1"/>
+                <xs:element name="password" type="Max5000Chars" minOccurs="0" maxOccurs="1"/>
```

Comment:
(i think already mentioned above) would `password_uri` be clearer?

Reply:
Password changes in inbc reverted.